src/shape.rs

```diff
@@ -213,6 +253,50 @@ fn shape_run(
     glyphs
 }
 
+#[cfg(feature = "swash")]
+fn shape_skip(
+    font_system: &mut FontSystem,
+    line: &str,
+    attrs_list: &AttrsList,
+    start_run: usize,
+    end_run: usize,
+) -> Vec<ShapeGlyph> {
+    let attrs = attrs_list.get_span(start_run);
+    let fonts = font_system.get_font_matches(attrs);
+
+    let default_families = [&attrs.family];
+    let mut font_iter = FontFallbackIter::new(font_system, &fonts, &default_families, Vec::new());
+
+    let font = font_iter.next().expect("no default font found");
+    let font_id = font.id();
+    let font = font.as_swash();
```

Comment:
Does it make sense for font fallback behaviour to be controlled by the same `Shaping` parameter? Would it make sense to expose this as a separate configuration option?

Reply:
It may. It's probably better to wait until we have a particular use case in mind, however.

I think the enum could be extended to accept a `fn` pointer at some point.

Reply:
I think it makes sense if we are to limit which Fonts the shaping could be used with. But this would also need to be Documented in the enum to give a clear idea of why this font or that font cant be loaded with this shaping algorithm.